content/source/docs/enterprise/private/encryption-password.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: "enterprise2"
+page_title: "Private Terraform Enterprise Encryption Password"
+sidebar_current: "docs-enterprise2-private-encryption-password"
+---
+
+# Private Terraform Enterprise Encryption Password
+
+During the installation of Private Terraform Enterprise, a password is used to encrypt sensitive information at rest. The default value is auto-generated, but we strongly suggest you create your own password.
+
+Be sure to retain the value, because you will need to use this
+password to restore access to the data in the event of a reinstall.
+
+The Encryption Password is used to protect the vault unseal
+key and root token when the internal Vault is used.
+It allows us to store those details in Postgresql,
+which makes vault “stateless” in that it only depends on details in Postgres and the encryption password.
+
+## Specifying the Encryption Password
+
+The password can be specified during an unattended installation with the installer option `enc_password` within the [application settings](#application-settings) file:
```

Comment:
This link is not correct, as the application settings are mentioned on a different page - can you make sure the link goes to that page?

Reply:
Fixed 👍